app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
Using an author alias instead of a user is a requirement of this project; we intentionally requested this so you can practice this. In the future, you may end up adding 3rd party (API, existing databases) resources to your already running application and 3rd party resources may have different kinds of data structure than you already have. In such cases, since you don't have control over 3rd party resources and refactoring your application can create bugs, you should use aliases to make 3rd party data compatible with your existing scheme.
I am so sorry that the second reviewer misinformed you about this.